### ISSUE TITLE
Transmitters > Soundboard > crash

### ISSUE THREAD

**Issue by Redouan29** [furrtek/portapack-havoc#110]
Build from source today. Cool new feature (last time I checked was a couple months ago).
But if you choose Transmitters > Soundboard , than the screen freezes and you forced to reset the device.

**Comment by furrtek** [comment]
How many files do you have in the /WAV directory ?
Looks like scan_root_files isn't happy, might be a RAM issue like #109

**Comment by chudgoo** [comment]
I too am seeing this. My SD card has about 10 files (all worked before upgrading to latest precompiled build)

**Comment by Redouan29** [comment]
I cleaned out the whole SD card, but still no entrance to the menu/option "soundboard".

**Comment by ImDroided** [comment]
Yep broke on my end also. I have to reset the unit upon clicking soundboard. I just compiled the latest

**Comment by F0GPI** [comment]
Wave file viewer freeze portpack also after choosing the wave file in sdcard. the bug can possibly arise from the sdcard gestion eventually? (just an hypothesis).

**Comment by furrtek** [comment]
RAM issue confirmed. Temporary fix for now is to get rid of the CTCSS option and limit files to 54 in 70c7646743ffc7af229bc8df8949b69feb23ecdc.

**Comment by furrtek** [comment]
Fixed in 96828262d712b08abb675b7c4424fec47d8ffb93.
